tests/functional/modules/test_zos_job_query_func.py

```diff
@@ -124,12 +124,14 @@ def test_zos_job_name_query_multi_wildcards_func(ansible_zos_module):
         for result in results.contacted.values():
             assert result.get("jobs")[0].get("ret_code").get("msg_code") == "0000"
             assert result.get("jobs")[0].get("ret_code").get("code") == 0
+            assert result.get("jobs")[0].get("execution_time") is not None
+            assert result.get("jobs")[0].get("system") is not None
+            assert result.get("jobs")[0].get("subsystem") is not None
 
             jobname = "HE*L*"
             qresults = hosts.all.zos_job_query(job_name=jobname, owner="*")
             for qresult in qresults.contacted.values():
                 assert qresult.get("jobs") is not None
-                assert qresult.get("jobs")[0].get("execution_time") is not None
```

Comment:
We should keep this assertion because we are implementing this enhancement in `zos_job_query`, is a good thing that you added those assertions inside the `zos_job_submit` response, keep them, but make sure that we validate them in `zos_job_query` response too.